bundle/src/main/java/com/adobe/acs/commons/mcp/impl/ControlledProcessManagerImpl.java

```diff
@@ -133,7 +133,15 @@ public ProcessDefinition findDefinitionByNameOrPath(String nameOrPath) throws Re
     }
 
     private ProcessDefinition findDefinitionByName(String name) throws ReflectiveOperationException {
-        Class defClass = Class.forName(name);
+        Class clazz;
+        try {
+            clazz = Class.forName(name);
```

Comment:
Well, technically Class.forName is faster for classes in the current bundle but I think you're right -- the catch block realistically is the only thing needed and class.forName can go away.